src/index.ts

```diff
@@ -230,6 +291,39 @@ export class CheckoutInstance {
     if (backgroundColor) {
       manifest.config!.backgroundColor = backgroundColor
     }
+    if (accentColor) {
+      manifest.config!.accentColor = accentColor
+    }
+    if (errorColor) {
+      manifest.config!.errorColor = errorColor
+    }
+    if (inputBackgroundColor) {
+      manifest.config!.inputBackgroundColor = inputBackgroundColor
+    }
+    if (inputBorderColor) {
+      manifest.config!.inputBorderColor = inputBorderColor
+    }
+    if (inputBorderRadius) {
+      manifest.config!.inputBorderRadius = inputBorderRadius
+    }
+    if (inputFloatingLabel) {
+      manifest.config!.inputFloatingLabel = inputFloatingLabel
+    }
+    if (buttonBackgroundColor) {
+      manifest.config!.buttonBackgroundColor = buttonBackgroundColor
+    }
+    if (buttonBorderRadius) {
+      manifest.config!.buttonBorderRadius = buttonBorderRadius
+    }
+    if (buttonBoxShadow) {
```

Comment:
I don't think this is enough, because if you specify `buttonBoxShadow: false` it won't enter the `if` and won't get passed on to the checkout iframe.

Reply:
True